keyboards/clueboard/66_hotswap/config.h

```diff
@@ -47,17 +47,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *  These options are also useful to firmware size reduction.
  */
 
-/* disable debug print */
 //#define NO_DEBUG
-
-/* disable print */
 //#define NO_PRINT
-
-/* disable action features */
 //#define NO_ACTION_LAYER
 //#define NO_ACTION_TAPPING
-//#define NO_ACTION_ONESHOT
-//#define NO_ACTION_MACRO
+#define NO_ACTION_ONESHOT
+#define NO_ACTION_MACRO
 //#define NO_ACTION_FUNCTION
```

Comment:
Wouldn't it be better to define `NO_ACTION_FUNCTION` than `NO_ACTION_ONESHOT`?  Since most (if not all) of the function stuff has keycodes, so action function isn't needed...

Reply:
I believe that `NO_ACTION_FUNCTION` is the one where layers stopped working entirely when I enabled it.

Reply:
That's ... odd.  I know that the `NO_ACTION_TAPPING` WILL do that, but `NO_ACTION_FUNCTION` should be just the older `fn_action` stuff.   

In fact, I have it defined in my userspace with no issues on any of my boards